app/src/main/res/layout/quest_bike_parking_capacity.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="bottom">
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_toLeftOf="@+id/capacityMultiplier"
+        android:id="@+id/capacityBike"
+        android:background="@drawable/ic_bicycle"/>
+
+    <TextView
+        android:id="@+id/capacityMultiplier"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
+        android:textSize="30dp"
+        android:padding="10dp"
+        android:text=" x " />
+
+    <EditText
+        android:layout_height="wrap_content"
+        android:layout_width="wrap_content"
+        android:ems="2"
+        android:layout_toRightOf="@+id/capacityMultiplier"
+        android:id="@+id/capacityInput"
+        android:inputType="number"
+        android:maxLines="1"
+        />
+
+</RelativeLayout>
```

Comment:
Better use ToEndOf/toStartOf etc. to make it more RTL compatible (for hopefully Arabian localization etc.)